### PR TITLE
fix: remove circle ci from pipeline

### PR DESCRIPTION
**Pull request by meta-dipanshu-sharma**
# COMPLETES #< [SPARK-474208](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-474208) >

## This pull request addresses

As we're using github actions for all PR checks so we want to remove circle ci from pipeline.

## by making the following changes

Removed `.circleci` folder which contains config file for circle ci and also removed other occurrences of circle ci from the code.

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by Kesari3008 — CHANGES_REQUESTED** [review]
Changes look fine.
I just saw one file: .coveralls.yml, not sure where it is getting used but it seems to contain token and service name is circleci so check once if we need that anymore or not
Also it looks like Amplify Console Web Preview is running from last 1 hour. So please check that also.

**Comment by meta-dipanshu-sharma** [comment]
> Changes look fine. I just saw one file: .coveralls.yml, not sure where it is getting used but it seems to contain token and service name is circleci so check once if we need that anymore or not Also it looks like Amplify Console Web Preview is running from last 1 hour. So please check that also.

@Kesari3008 `coveralls` is a code coverage service that helps tracking the quality/coverage of code after each PR. I checked the commit history and seems like it was added with the first ever commit to js sdk 6 years back. We're not using it anymore so I can remove it in this PR but just want to confirm once with @arun3528. 

Regarding amplify check, I don't have access to aws as my password got expired so I have asked Robin to reset it and then only I'll be able to check the issue with it.

File: karma-ng.conf.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ function makeConfig(packageName, argv) {
     recordScreenshots: true,
   };
 
-  if (process.env.COVERAGE || process.env.CIRCLECI || process.env.CI) {
```
Comment (arun3528) [inline karma-ng.conf.js:160]: do we know why this was getting used in the first place ?

Comment (arun3528) [reply karma-ng.conf.js:160]: do we have to intoduce something for workflow as well

**Review by arun3528 — APPROVED** [review]
LGTM 

lets remove the old sauce coverage

File: karma-ng.conf.js (hunk 1 above)
Comment (meta-dipanshu-sharma) [reply karma-ng.conf.js:160]: discussed offline.

**Comment by meta-dipanshu-sharma** [comment]
@Kesari3008: I removed the Coveralls config after discussing it with Arun, as we're not using it for our PRs. Regarding the Amplify check, it ran successfully after I re-ran it.

**Pull request merged**